x/staking/spec/09_client.md

```diff
@@ -0,0 +1,612 @@
+```
+
+Example Output:
+
+```bash
+pagination:
+  next_key: null
+  total: "0"
+unbonding_responses:
+- delegator_address: cosmos1gghjut3ccd8ay0zduzj64hwre2fxs9ld75ru9p
+  entries:
+  - balance: "52000000"
+    completion_time: "2021-11-02T11:35:55.391594709Z"
+    creation_height: "55078"
+    initial_balance: "52000000"
+  validator_address: cosmosvaloper1t8ehvswxjfn3ejzkjtntcyrqwvmvuknzmvtaaa
+
+```
+
+#### unbonding-delegations-from
+
+The `unbonding-delegations-from` command allow user to query delegations that are unbonding _from_ a validator.
+
+Usage:
+
+```bash
+simd query staking unbonding-delegations-from [validator-addr] [flags]
+```
+
+Example:
+
+```bash
+simd query staking unbonding-delegations-from cosmosvaloper1gghjut3ccd8ay0zduzj64hwre2fxs9ldmqhffj
+```
+
+Example Output:
+
+```bash
+pagination:
+  next_key: null
+  total: "0"
+unbonding_responses:
+- delegator_address: cosmos1qqq9txnw4c77sdvzx0tkedsafl5s3vk7hn53fn
+  entries:
+  - balance: "150000000"
+    completion_time: "2021-11-01T21:41:13.098141574Z"
+    creation_height: "46823"
+    initial_balance: "150000000"
+  validator_address: cosmosvaloper1gghjut3ccd8ay0zduzj64hwre2fxs9ldmqhffj
+- delegator_address: cosmos1peteje73eklqau66mr7h7rmewmt2vt99y24f5z
+  entries:
+  - balance: "24000000"
+    completion_time: "2021-10-31T02:57:18.192280361Z"
+    creation_height: "21516"
+    initial_balance: "24000000"
+  validator_address: cosmosvaloper1gghjut3ccd8ay0zduzj64hwre2fxs9ldmqhffj
+```
+
+#### validator
+
+The `validator` command allow user to query details about an individual validator.
+
+Usage:
+
+```bash
+simd query staking validator [validator-addr] [flags]
+```
+
+Example:
+
+```bash
+simd query staking validator cosmosvaloper1gghjut3ccd8ay0zduzj64hwre2fxs9ldmqhffj
+```
+
+Example Output:
+
+```bash
+commission:
+  commission_rates:
+    max_change_rate: "0.020000000000000000"
+    max_rate: "0.200000000000000000"
+    rate: "0.050000000000000000"
+  update_time: "2021-10-01T19:24:52.663191049Z"
+consensus_pubkey:
+  '@type': /cosmos.crypto.ed25519.PubKey
+  key: sIiexdJdYWn27+7iUHQJDnkp63gq/rzUq1Y+fxoGjXc=
+delegator_shares: "32948270000.000000000000000000"
+description:
+  details: Witval is the validator arm from Vitwit. Vitwit is into software consulting
+    and services business since 2015. We are working closely with Cosmos ecosystem
+    since 2018. We are also building tools for the ecosystem, Aneka is our explorer
+    for the cosmos ecosystem.
+  identity: 51468B615127273A
+  moniker: Witval
+  security_contact: ""
+  website: ""
+jailed: false
+min_self_delegation: "1"
+operator_address: cosmosvaloper1gghjut3ccd8ay0zduzj64hwre2fxs9ldmqhffj
+status: BOND_STATUS_BONDED
+tokens: "32948270000"
+unbonding_height: "0"
+unbonding_time: "1970-01-01T00:00:00Z"
+```
+
+#### validators
+
+The `validators` command allow user to query details about all validators on a network.
+
+Usage:
+
+```bash
+junod query staking validators [flags]
```

Comment:
Let's use `simd` to be consistent with other module examples 
```suggestion
simd query staking validators [flags]
```

Reply:
Resolved in this [commit](https://github.com/cosmos/cosmos-sdk/pull/10306/commits/951a9c1d166f3199cc95c23ed0864f5e8674b42e).

Reply:
@anilCSE  Added the CLI client tx commands docs as well(Please review them). Will add grpc and rpc Client docs in further commits.